kong/tools/responses.lua

```diff
@@ -142,9 +141,11 @@ local function send_response(status_code)
                                   {message = content})
       if not encoded then
         ngx.log(ngx.ERR, "[admin] could not encode value: ", err)
-      end
 
-      ngx.say(encoded)
+      else
+        ngx.header["Content-Type"] = "application/json; charset=utf-8"
+        ngx.say(encoded)
```

Comment:
:+1: 

We may need some unit tests for this (they exist for this module already)

Reply:
Ah, my bad, I see you already added it :)